packages/execute-driver-plugin/test/e2e/plugin.e2e.spec.js

```diff
@@ -2,30 +2,38 @@
 
 import path from 'path';
 
-import { e2eSetup as _e2eSetup } from '@appium/base-plugin/build/test/helpers';
-import { remote as wdio } from 'webdriverio';
-import { W3C_ELEMENT_KEY, MJSONWP_ELEMENT_KEY } from '../../lib/execute-child';
-import { fs } from '@appium/support';
+import {e2eSetup as _e2eSetup} from 'appium/plugin';
```

Comment:
was this import moved to the package index?

Reply:
no, this is wrong. it will be moved to test-support later